bench/DataInputOutputStreamBench.java

```diff
@@ -0,0 +1,58 @@
+package benchmark;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.DataOutputStream;
+import java.io.DataInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import com.sun.cldchi.jvm.JVM;
+
+public class DataInputOutputStreamBench {
+  void writeUTF(OutputStream out) throws IOException {
+      DataOutputStream dos = new DataOutputStream(out);
+
+      for (int i = 0; i < 25000; i++) {
+          dos.writeUTF("abcdefghilmnopqrstuvzABCDEFGHILMNOPQRSTUVZabcdefghilmnopqrstuvzABCDEFGHILMNOPQRSTUVZ");
```

Comment:
I just copied & pasted the Italian alphabet :smile: 

I don't recall why I made a 25k loop instead of a 50k loop, so I've changed it with 17dd80b.